app/src/androidTest/java/org/fossasia/susi/ai/helper/UtilsTest.kt

```diff
@@ -0,0 +1,27 @@
+package org.fossasia.susi.ai.helper
+
+import android.support.test.runner.AndroidJUnit4
+import android.util.Patterns
+import junit.framework.Assert.assertTrue
+import org.fossasia.susi.ai.rest.responses.susi.SkillData
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(AndroidJUnit4::class)
+class UtilsTest {
+
```

Comment:
@iamareebjamal since there are no Unit tests in the app right now , on running the tests ClassNotFound is shown each time, even running an empty test gives this error.

Reply:
Uhh. This is androidTest. Please add in the unit test directory